lib/liquid_swipe.dart

```diff
@@ -63,15 +61,21 @@ class _LiquidSwipe extends State<LiquidSwipe> with TickerProviderStateMixin {
   AnimatedPageDragger
       animatedPageDragger; //When user stops dragging then by using this page automatically drags.
 
-  int activePageIndex = 0; //active page index
+  int _activePageIndex = 0; //active page index
```

Comment:
Hi, I am not quite sure here, like you have changes the variable name but used previous name in line 197.